wiki/Medals/Legacy_beatmap_packs/de.md

```diff
@@ -0,0 +1,406 @@
+# Originale Beatmap-Pakete
+
+Beatmap-Pakete sind ein integraler Bestandteil der osu!-Community seit [ihrer ersten Bekanntmachung](https://osu.ppy.sh/community/forums/topics/1853) von ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) vor langer Zeit im Jahr 2008, in dem einige der Pakete mit bestimmten [Errungenschaften](/wiki/Medals) verbunden waren. Als osu! und seine Community jedoch über die Jahre wuchsen, hat sich das Mapping-Meta im Allgemeinen seit seinen Anfängen stark verändert und es wurde — durch große Nachfrage der Öffentlichkeit — [am 16. Januar 2016](https://osu.ppy.sh/home/news/2016-01-18-osuweekly-44) eine umfassende Überarbeitung bei zahlreichen Beatmap-Paketen vorgenommen, um diese in ihrer Gesamtheit auf dem neusten Stand zu halten.
```

Comment:
```suggestion
Beatmap-Pakete sind ein integraler Bestandteil der osu!-Community seit [ihrer ersten Bekanntmachung](https://osu.ppy.sh/community/forums/topics/1853) von ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) vor langer Zeit im Jahr 2008, in dem einige der Pakete mit bestimmten [Errungenschaften](/wiki/Medals) verbunden waren. Als osu! und seine Community jedoch über die Jahre wuchsen, hat sich die Mapping-Meta im Allgemeinen seit ihren Anfängen stark verändert und es wurde — durch große Nachfrage der Öffentlichkeit — [am 16. Januar 2016](https://osu.ppy.sh/home/news/2016-01-18-osuweekly-44) eine umfassende Überarbeitung bei zahlreichen Beatmap-Paketen vorgenommen, um diese in ihrer Gesamtheit auf dem neusten Stand zu halten.
```
Pretty sure it should be `die Meta` though I couldn't find any German definitions for this, but at least from my experience this is the most common.

Reply:
My initial thought was to go with the neutral article, but this is fine as well.

After some small internet search ~~(and excluding the new facebook lol)~~, I've found the following things:
- [A word definition on wiktionary](https://de.wiktionary.org/wiki/meta-), however this isn't helpful here
- Deepl actually translates `the game meta` with `die Spiel-Meta`

Reply:
Yeah i found the wiktionary one as well and Duden has it too, but both are for prefixes, so not really helpful.